src/tools/build-manifest/src/main.rs

```diff
@@ -23,6 +23,10 @@ static HOSTS: &'static [&'static str] = &[
     "mips64-unknown-linux-gnuabi64",
     "mips64el-unknown-linux-gnuabi64",
     "mipsel-unknown-linux-gnu",
+    "mipsisa32r6-unknown-linux-gnu",
```

Comment:
This is only necessary if the target is available thought rustup.

Reply:
rustup looks so great.

If possible, I wish that we can support r6 with it.
How these binary is built? by cross or qemu?